### PR TITLE
Support last FluentKit beta 5

### PR DESCRIPTION
**Pull request by Maxim-Inv**

### CODE REVIEW

**Comment by MihaelIsaev** [comment]
Tests failed, force unwrapping and using description instead of direct value is not a good idea cause force unwrapping may fail the whole server and description may be changed as well.

**Comment by Maxim-Inv** [comment]
Tests failed because swift:5.1 docker used. How to change to use swift 5.2?

**Comment by MihaelIsaev** [comment]
https://github.com/MihaelIsaev/SwifQLVapor/blob/master/.github/workflows/test.yml

**Comment by Maxim-Inv** [comment]
This description is an internal ivar of FieldKey 
https://github.com/vapor/fluent-kit/blob/master/Sources/FluentKit/Properties/FieldKey.swift#L7

**Comment by MihaelIsaev** [comment]
Do they use this `description` variable to access the name of the field? Cause basically it is just to conform to `CustomStringConvertible` and it may return anything to describe the object and may be changed at any time.

**Comment by Maxim-Inv** [comment]
I saw it and took it from here - https://github.com/vapor/fluent-postgres-driver/blob/master/Sources/FluentPostgresDriver/PostgresRow%2BDatabase.swift#L46

**Comment by MihaelIsaev** [comment]
Also could you please change swift-tools-version to 5.2 in the `Package.swift`

**Comment by MihaelIsaev** [comment]
> I saw it and took it from here - https://github.com/vapor/fluent-postgres-driver/blob/master/Sources/FluentPostgresDriver/PostgresRow%2BDatabase.swift#L46

Ok then yeah it looks like it is the official way to get the name of the field

**Comment by Maxim-Inv** [comment]
Of course

**Comment by MihaelIsaev** [comment]
Hmm that's weird that tests are failing again..

File: Package.swift [hunk 1]
```diff
@@ -18,7 +18,7 @@ let package = Package(
         .package(url: "https://github.com/vapor/vapor.git", from: "4.0.0-beta.2"),
         .package(url: "https://github.com/vapor/fluent.git", from: "4.0.0-beta.2"),
         .package(url: "https://github.com/vapor/fluent-postgres-driver.git", from: "2.0.0-beta.2"),
-        .package(url: "https://github.com/vapor/fluent-mysql-driver.git", from: "4.0.0-beta.2"),
```
Comment (MihaelIsaev) [inline Package.swift]: It should not affect tests. it'd be better to uncomment it and change back swift-tools-version to 5.1

Comment (Maxim-Inv) [reply Package.swift]: Tried, just for test

**Comment by MihaelIsaev** [comment]
Ok now it is available in `2.1.0` tag, thanks for the contribution! 🤝

**Pull request merged**